### PR TITLE
docs: Add OpenEBS troubleshooting docs

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: docs/how-to-guides/troubleshoot-openebs.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+This guide enlists various issues with OpenEBS and steps to mitigate those problems.
+
+## Prerequisites
+
+* A Kubernetes cluster accessible via `kubectl`.
+
+* [OpenEBS component](../../configuration-reference/components/openebs-operator) installed.
+
+## ISCSI volume mount issue
```
Comment (invidian) [inline docs/how-to-guides/troubleshoot-openebs.md:16]:
```suggestion
## iSCSI volume mount issue
```

File: docs/how-to-guides/troubleshoot-openebs.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Acquire root credentials:
+
+```bash
+sudo -i
+```
+
+Create the `iscsid` service dropin and restart the service with new config, copy paste the following commands:
+
+```bash
+mkdir /run/systemd/system/iscsid.service.d
```
Comment (invidian) [inline docs/how-to-guides/troubleshoot-openebs.md:63]: Is using `/run` here on purpose? If so, why?

File: docs/how-to-guides/troubleshoot-openebs.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+> **NOTE**: If any of the following commands fail then you might not be running a self-hosted
+> Kubelet in your cluster.
+
+#### Step 1.1: Update the image
+
+```bash
+kubectl -n kube-system set image ds kubelet kubelet=quay.io/kinvolk/kubelet:v1.19.4
+```
+
+#### Step 1.2: Remove the iscsi mount
```
Comment (invidian) [inline docs/how-to-guides/troubleshoot-openebs.md:36]:
```suggestion
#### Step 1.2: Remove the iSCSI mount
```

**Review by invidian — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/troubleshoot-openebs.md (hunk 2 above)
Comment (surajssd) [reply docs/how-to-guides/troubleshoot-openebs.md:63]: I found all the other places to create a systemd dropin as read-only. I would like to make it persistent. Is there a more permanent way of doing it?

Comment (invidian) [reply docs/how-to-guides/troubleshoot-openebs.md:63]: Well, `/run` is definitely not permanent :smile: Put them in `/etc/systemd` of course :)

Comment (surajssd) [reply docs/how-to-guides/troubleshoot-openebs.md]: Done. I don't know but I got read-only FS error when doing it in /etc, but now it seems to work. Not sure what went wrong when I tried it for the first time.

**Review by invidian — APPROVED** [review]
LGTM. Thanks @surajssd

**Pull request merged**